.kitchen.yml

```diff
@@ -36,7 +37,7 @@ default: &default
 suites:
   - name: default
```

Comment:
This file is feeling over-complicated...do we need all this includes/excludes logic still?